omero/developers/Web/CreateApp.rst

```diff
@@ -11,144 +11,43 @@ Getting set up
-
-    # Location within Docker instance we want to mount the config.
-    $ docker_config=/opt/omero/web/config/config.omero
-
-    # Run docker container.
-    $ docker run -it --rm -e OMEROHOST=$host -p 4080:4080 -v $appdir:$docker_appdir -v $config:$docker_config openmicroscopy/omero-web-standalone
-
-This will run Docker in the foreground, showing the output in your terminal and allowing you to
-kill the container with Ctrl-C. You should see the following lines in the output, indicating
-that OMERO.web is starting and the static files from your app are being included.
-
-::
-
-    ...
-    Copying '/opt/omero/web/venv3/lib/python3.6/site-packages/minimal_webapp/static/minimal_webapp/app.css'
-    Copying '/opt/omero/web/venv3/lib/python3.6/site-packages/minimal_webapp/static/minimal_webapp/app.js'
-    ...
-    Starting OMERO.web...
-
-Now go to `http://localhost:4080/minimal_webapp/ <http://localhost:4080/minimal_webapp/>`_
-in your browser.
-You should be redirected to the login screen and then back to the ``minimal_webapp``
-page which will display your Name and list your Projects.
+Please follow the instructions there to install `cookiecutter`, create your app
+and run it within your `omero-web` python environment.
 
 Exploring the app
 -----------------
 
-The `minimal_webapp code <https://github.com/ome/omero-web-apps-examples/tree/master/minimal-webapp>`_
-is well-documented to explain how the app is working.
+The newly created app template is well-documented to explain how the app is working.
 Briefly, the app supports a single URL defined in
-``minimal_webapp/urls.py`` which maps to the ``index`` function
-within ``minimal_webapp/views.py``. This uses the connection to
+``urls.py`` which maps to the ``index`` function
+within ``views.py``. This uses the connection to
 OMERO, ``conn`` to load the current user's name and passes this
 to the ``index.html`` template to render the page.
 
 This page also includes the static ``app.js`` and ``app.css`` files.
 JavaScript is used to load Projects from the :doc:`/developers/json-api` and
 display them on the page.
 
-Create an app from the template example
----------------------------------------
-
-If you want to create your own app, you can use the example
-as a template.
-
-Go to the template repository
-`omero-web-apps-examples <https://github.com/will-moore/omero-web-apps-examples>`_.
-Click 'Use this template' as `described here
-<https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template>`_
-and choose a name for your new repo, for example ``my_app``.
-
-Go to the directory where you want your app to live and clone it.
-Then run as above with Docker or locally-installed OMERO.web, making sure
-that your app can be imported as before.
-
-::
-
-    $ git clone https://github.com/<username>/my_app
-    $ cd my_app
-
-    # Then run as above...
-
+These represent the 2 main options you have for loading data from OMERO and displaying
```

Comment:
What does "These" refer to here ? To ``app.js`` and ``app.css`` ? If so, then there should not be an empty line between this and previous paragraph. If not, I am lost in the text :)

Reply:
Maybe the "These ...2 main options..." refer to the kinda "Django" option and the option using JavaScript. Then, the two options correspond to the two paragraphs above ? If so, it would be helpful to add word "Django" to the first paragraph in order not to loose the context.

Reply:
Thanks @pwalczysko - reformatted this section in 075bf619